backend-api/api/v1.0.yaml

```diff
@@ -68,6 +68,22 @@ paths:
         "default":
           $ref: "#/responses/ErrorSet"
   #-----------------------------------------------------------------------------
+  /me/team:
+    x-swagger-router-controller: users
+    put:
+      description: Join user to team.
+      operationId: modifyUserTeam
+      x-security: [bearer, ignore-terms-lock]
```

Comment:
Please double-check that this endpoint should be accessible when the user hasn't accepted terms of use. If not, remove the `ignore-terms-lock` flag from `x-security`.

Reply:
`ignore-terms-lock` should be here. User will not be able to interact with the app until he accepts the terms.

Reply:
Just making sure, `ignore-terms-lock` **skips** the terms check, so if it's **present** the user **can** change their team before having accepted terms. I know it can be confusing since it's a negative, sorry. :)